x-pack/plugins/rollup/server/routes/api/jobs.js

```diff
@@ -107,9 +108,17 @@ export function registerJobsRoute(server) {
     handler: async (request) => {
       try {
         const { jobIds } = request.payload;
+        // For our API integration tests we need to wait for the jobs to be stopped
+        // in order to be able to delete them sequencially.
+        const { waitForCompletion } = request.query;
         const callWithRequest = callWithRequestFactory(server, request);
-        return await Promise.all(jobIds.map(id => callWithRequest('rollup.stopJob', { id })))
+
+        const stopRollupJob = id => callWithRequest('rollup.stopJob', { id, waitForCompletion: waitForCompletion === 'true' });
```

Comment:
@cjcenizal @jen-huang Do you know how I can provide the `waitForCompletion` param in the `callWithRequest()` call? On L94 of 'rollup/server/client/elasticsearch_rollup.js' above I defined the `params` config, but I'm not sure how to provide it. Or is it automatically done through the `request` object?

Reply:
I got an answer from Tomas (della Vedova). The current implementation is correct. All the properties of the object provided in the `callWithRequest` will be automatically mapped to either the `params` object, either the `req` object in the elasticsearch `ca()` method.